src/routes/register.ts

```diff
@@ -0,0 +1,28 @@
+import type { RequestEvent } from '@sveltejs/kit';
+
+import { getLocale } from '$lib/utilities/i18n.utilities';
+
+// Supported locales
+import ca_en from '$locales/ca/en/index.json';
+import ca_fr from '$locales/ca/fr/index.json';
+import us_en from '$locales/us/en/index.json';
+import us_es from '$locales/us/es/index.json';
+
+const locales = {
+  ca_en,
```

Comment:
I was slightly surprised to see the locales hard coded within the app - might this increase the "cost" of adding more locales later once the app is in production? My thinking is of the effort needed to support a new language with a code and config change, as opposed to just a config change e.g. drop in a new language file with a filename matching the locale. What are your thoughts about this?

Reply:
I was originally using purely directory/file based methods, but I ended up being completely obstructed by the affect this had in deployment. The locales weren't being deployed because nothing was referencing them in the app. E.g.: I was using an `async import` like this: 

```
async import(`locales/${localDir}/index.json`);
```

The problem is when the above gets compiled for deployment, all of the local files are not "statically" referenced, so they are not deployed. If I can figure out how to get around that, then we can revisit it. But, to no block the rest of the project, I needed a way to get around it and move on.

Reply:
If we can revisit the ability to use `async imports`, I too would prefer using that over this approach.